admin/server/deployment.go

```diff
@@ -182,12 +182,18 @@ func (s *Server) GetDeployment(ctx context.Context, req *adminv1.GetDeploymentRe
 	}
 
 	var attr map[string]any
+	var restrictResources bool
+	var resources []database.ResourceName
 	if req.For == nil {
 		if claims.OwnerType() == auth.OwnerTypeUser {
 			attr, err = s.jwtAttributesForUser(ctx, claims.OwnerID(), proj.OrganizationID, permissions)
 			if err != nil {
 				return nil, err
 			}
+			restrictResources, resources, err = s.getResourceRestrictionsForUser(ctx, proj.ID, claims.OwnerID())
```

Comment:
Is there a reason for two separate calls here instead of using `getAttributesAndResourceRestrictionsForUser`?